lib/v2/design/app_theme.dart

```diff
@@ -168,3 +172,67 @@ extension CustomStyles on TextTheme {
   TextStyle get headline7Green =>
       headline7.copyWith(fontSize: 22, fontWeight: FontWeight.w500, color: AppColors.green3);
 }
+
+class SeedsInputDecorationTheme {
```

Comment:
To point to our conversation in whatsap. 
This could be its own pr. 

reviewed and Tested in isolation independent of the main ticket. Etc